src/ipfs-on-path/index.js

```diff
@@ -1,123 +1,74 @@
+      logger.error(`[ipfs on path] ${err.toString()}`)
     }
 
-    if (newValue === true) {
-      if (showDialog({
-        title: i18n.t('enableIpfsOnPath.title'),
-        message: i18n.t('enableIpfsOnPath.message'),
-        buttons: [
-          i18n.t('enableIpfsOnPath.action'),
-          i18n.t('cancel')
-        ]
-      }) !== 0) {
-        // User canceled
-        return
-      }
-
-      return run('install')
+    try {
+      unlinkSync(join(app.getPath('home'), './.ipfs-desktop/IPFS_PATH').replace('app.asar', 'app.asar.unpacked'))
+      unlinkSync(join(app.getPath('home'), './.ipfs-desktop/IPFS_EXEC').replace('app.asar', 'app.asar.unpacked'))
+    } catch (err) {
+      // Weird, but not worth bothering.
+      logger.error(`[ipfs on path] ${err.toString()}`)
     }
 
-    if (showDialog({
-      title: i18n.t('disableIpfsOnPath.title'),
-      message: i18n.t('disableIpfsOnPath.message'),
+    logger.info('[ipfs on path] uninstalled')
+
+    showDialog({
+      title: 'Command Line Tools Uninstalled',
+      message: 'Command Line Tools via IPFS Desktop have been deprecated in February 2021. They have now been uninstalled. Please refer to https://docs.ipfs.io/install/command-line/ if you need to use ipfs from the command line.',
       buttons: [
-        i18n.t('disableIpfsOnPath.action'),
-        i18n.t('cancel')
+        i18n.t('close')
       ]
-    }) !== 0) {
-      // User canceled
-      return
-    }
-
-    return run('uninstall')
-  })
-
-  firstTime()
-}
-
-module.exports.CONFIG_KEY = CONFIG_KEY
-
-async function firstTime () {
-  // Check if we've done this before.
-  if (store.get(CONFIG_KEY, null) !== null) {
-    logger.info('[ipfs on path] no action taken')
-    return
-  }
-
-  if (which.sync('ipfs', { nothrow: true }) !== null) {
-    // ipfs already exists on user's system so we won't take any action
-    // by default. Doesn't try again next time.
-    store.set(CONFIG_KEY, false)
-    return
+    })
   }
 
-  // Tries to install ipfs-on-path on the system. It doesn't try to elevate
-  // to sudo so the user doesn't get annoying prompts when running IPFS Desktop
-  // for the first time. Sets the option according to the success or failure of the
-  // procedure.
-  try {
-    const res = await run('install', { trySudo: false, failSilently: true })
-    store.set(CONFIG_KEY, res)
-  } catch (err) {
-    logger.error(`[ipfs on path] unexpected error while no-sudo install: ${err.toString()}`)
-    store.set(CONFIG_KEY, false)
-  }
+  store.delete(CONFIG_KEY)
 }
 
-async function runWindows (script, { failSilently }) {
-  return new Promise(resolve => {
+async function uninstallWindows () {
+  return new Promise((resolve, reject) => {
     execFile('powershell.exe', [
       '-nop', '-exec', 'bypass',
       '-win', 'hidden', '-File',
-      join(__dirname, `scripts/${script}.ps1`).replace('app.asar', 'app.asar.unpacked')
+      join(__dirname, 'scripts/uninstall.ps1').replace('app.asar', 'app.asar.unpacked')
```

Comment:
Similar note as one for `uninstall.js` – I am not sure if running the `uninstall.ps1` script is safe:

https://github.com/ipfs/ipfs-desktop/blob/b73c1cc4ee6bc8efa9afdce13fa11ee8bb038f0c/src/ipfs-on-path/scripts/uninstall.ps1#L1-L3

Where does the `bin-win` come from? Is it something created by electron-builder for Windows?

 If I understand correctly, it is impossible to distinguish between user who enabled in before February 2021, and users who manually added the binary to their PATH? 
This means we could break someone's setup.

Let's leave `PATH` as-is to minimize breakage of power users.

Reply:
`bin-win` is given by _us_: https://github.com/ipfs/ipfs-desktop/tree/main/src/ipfs-on-path/scripts/bin-win

I don't think anyone would've added that to the `PATH` manually. At least, they shouldn't. This file is also removed on this PR. So... even if they have it in the `PATH`, it won't work.

This script was already here. They're not new scripts and they've worked until now. I was just planning on reusing them.

Reply:
Thanks for explaining, sgtm!